OPHD/Technology/TechnologyCatalog.cpp

```diff
@@ -172,16 +172,17 @@ namespace
 		for (auto category = &node; category; category = category->nextSiblingElement())
 		{
 			const auto attributes = NAS2D::attributesToDictionary(*category);
-			TechnologyCatalog::Category newCategory{attributes.get<int>("icon_index"), attributes.get("name"), {}};
+			const std::string name = attributes.get("name");
 
 			for (const auto& cat : categories)
 			{
-				if (cat.name == newCategory.name)
+				if (cat.name == name)
 				{
-					throw std::runtime_error("TechnologyReader: Category redefinition '" + newCategory.name + "'" + nodeAtString(*category));
+					throw std::runtime_error("TechnologyReader: Category redefinition '" + name + "'" + nodeAtString(*category));
 				}
 			}
-			newCategory.technologies = readTechnologiesInCategory(*category);
+			
+			const TechnologyCatalog::Category newCategory{attributes.get<int>("icon_index"), name, readTechnologiesInCategory(*category)};
 			categories.push_back(newCategory);

```

Comment:
Hmm, could even use `emplace_back` here, so the `Category` is constructed in-place in the `std::vector`:
```cpp
categories.emplace_back(attributes.get<int>("icon_index"), name, readTechnologiesInCategory(*category));
```

Reply:
Also, the use of `const` here takes away from the possibility of using `std::move` (or rather, you would just get a copy if you tried). Though `emplace_back` is better, since then there's no need to move/copy.